src/lib/Scenes/Show/Components/Artists/__tests__/Artists-tests.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import { graphql } from "react-relay"
+
+import { MockRelayRenderer } from "../../../../../tests/MockRelayRenderer"
+import { renderUntil } from "../../../../../tests/renderUntil"
+import { ShowFixture } from "../../../__tests__/fixtures"
+
+import { InvertedButton } from "lib/Components/Buttons"
+import ListItemButton from "lib/Components/Buttons/InvertedButton"
+import { ArtistListItem } from "../Components/ArtistListItem"
+import { Artists, ArtistsContainer } from "../index"
+
+jest.unmock("react-relay")
+import relay from "react-relay"
+
+relay.commitMutation = jest.fn()
+
+const renderTree = () =>
+  renderUntil(
+    wrapper => {
+      return wrapper.find(ArtistListItem).length > 0
+    },
+    <MockRelayRenderer
+      Component={ArtistsContainer}
+      query={graphql`
+        query ArtistsTestsQuery {
+          show(id: "anderson-fine-art-gallery-flickinger-collection") {
+            ...Artists_show
+          }
+        }
+      `}
+      mockResolvers={{
+        Show: () => ShowFixture,
+        Artist: () => ({ name: "Hans Hofmann", is_followed: false, id: "hans-hofmann" }),
```

Comment:
Is there a better way to mock the list of artists in a show? It seems like I'm only getting 2 artists in the rendered component, when there's many more in `ShowFixture`, which makes me think something akin to the "Hello, world!" stub data is being generated.

Reply:
As of now no, it's a limitation from relay-mock-resolver not mocking GraphQLList types properly when nested in a parent type.
We have to do that for connection types as well.